### ISSUE TITLE
[Docs] Document where to download the latest release version for any given network

### ISSUE THREAD

**Issue by davidiw** [aptos-labs/aptos-core#6704]
# Aptos Documentation Issue

It isn't obvious where to download the latest release version for any given network:

How does one identify the correct images to download for any given network?
https://github.com/aptos-labs/aptos-core/releases shows that 1.2.5 released
https://hub.docker.com/r/aptoslabs/validator/tags?page=1&name=1.2.5 shows several variants
https://github.com/aptos-labs/aptos-networks/tree/main/mainnet shows a long time ago
There's also this slack: https://aptos-org.slack.com/archives/C045Y5DP61J/p1675356115575409

Then @sherry-x mentioned it is in the release channel on discord.

It would be great to have this early on in the node documentation. We should also consider if we want to add this information to the mainnet networks readme.

**Comment by clay-aptos** [comment]
Adding @davidiw, @sherry-x and @thepomeranian as this is clearly as much a process as documentation issue. See this PR for progress:
https://github.com/aptos-labs/aptos-core/pull/6657

And related work on the process in:
https://github.com/aptos-labs/aptos-core/issues/6373

Staged at:
https://deploy-preview-6657--aptos-developer-docs.netlify.app/aptos-releases
https://deploy-preview-6657--aptos-developer-docs.netlify.app/nodes/aptos-deployments

In short, we need to ID the flow of this information as I am seeing weeks-long gaps between releases:
https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.2.5

And corresponding announcements in Discord:
https://discord.com/channels/945856774056083548/1042502400507916349/1077639363141312522

What is the ideal flow here? Who can Frances and I work with the lay out what information is available when.

**Comment by thepomeranian** [comment]
I think the index of latest releases should be with "Latest Releases" and not under Aptos Deployment. For Aptos Deployments link, I would recommend a link out to the Latest Releases table sorting by latest with the filter (mainnet,testnet,devnet). 

https://excalidraw.com/#room=47432cee87433528a055,L6H6hipPJBrMl897iKE1fQ

**Comment by clay-aptos** [comment]
Hi @thepomeranian, please help me gather the necessary information from @sherry-x to populate the table shown in:
https://deploy-preview-6657--aptos-developer-docs.netlify.app/aptos-releases

It sounds like we need to depict versions for:
* Node
* CLI
* Framework

By Network:
* Mainnet
* Testnet
* Devnet

Let's see if we can first populate Node table in the page above. We can then make it repeatable for others. Thanks!

**Comment by thepomeranian** [comment]
Not sure if this is what you mean:

mainnet:
* Node: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-node-v1.2.6
* CLI: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-cli-v1.0.6
* Framework: https://github.com/aptos-labs/aptos-core/releases/tag/aptos-framework-v1.2.0

We manage testnet and devnet right now so we'll not providing links to them just the status. Please feel free to correct me if I'm wrong.
* Testnet v1.6
* Devnet 1.3 (coming today)
